leptos_hotkeys/Cargo.toml

```diff
@@ -14,6 +14,7 @@ leptos = "0.6"
 log = { version = "0.4", optional = true }
 wasm-bindgen = "0.2"
 web-sys = "0.3"
+wasm-keys = { git = "https://github.com/gaucho-labs/wasm-keys" }
```

Comment:
This is really unsafe. Would you mind to release a version of wasm-keys?

Reply:
Yes -- sorry I was going to do this during the weekend